data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,328 @@ mission "Coalition: Outskirts Gauger Ready"
+			choice
+				`	"I'm interested. Wait here, I'll go buy myself a copy."`
+					goto "buy her book"
+				`	"Why is it a story about the Quarg? Do you hate the Quarg or something?"`
+					goto "do you hate the quarg"
+				`	"Cool. How has life been lately as an author?"`
+			label "life as an author sucks"
+			`	"Bad," says Anna. "I've never had a bestseller or even a book that surpasses 10,000 copies sold."`
+			choice
+				`	"What are your books about?"`
+					goto "what are her books about"
+				`	"Well, that's gotta hurt. I hope you find success in your life one day.`
+					decline
+			label "buy her book"
+			action
+				set "beyond the hyperdrive book"
+				payment -60
+			`	You enter the bookstore, ask the employee for one of Anna's books, and purchase it for 60 credits.`
+			choice
+				`	(Now go to Anna and get it signed.)`
+				`	(Leave.)`
+					decline
+			`	You move to the table where Anna is sitting. She looks visibly happy as she grabs your book, takes out a pen, and signs it. "Thanks for buying one of my books. I don't get much sales and people like you mean the world to me."`
+			choice
+				`	"I appreciate it. How has life as an author been though?"`
+					goto "life as an author sucks"
+				`	"Thanks for signing my copy. I hope you find success in your life. Goodbye."`
+					decline
+			label "do you hate the quarg"
+			`	"I have a lot of problems with them. You see, there are so many systems we can't reach and the Quarg have the secrets to reaching them. And yet they won't give humanity the secret. They are trying to restrict us. Keeping us all locked in cages! So, I wrote 'Beyond the Hyperdrive' to share my feelings about the Quarg. Humanity will one day succeed! We will surpass them and make humanity even greater beyond its wildest dreams!"`
+			choice
+				`	(Tell her about the Coalition.)`
+			label "tell her about the coalition"
+			`	You check your surroundings, looking around to ensure you are not being listened to. Once you confirm that your conversation will be private, you begin speaking.`
+			`	"I know of a place called the Coalition. They're a group of alien races. Maybe you could go there and find inspiration for your next novel, or sell your books there? I bet they'd love to see your writings!"`
+			`	"Wait, really?" asks Anna.`
+			`	You continue to explain the Coalition to Anna for a while, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them.`
+			choice
+				`	"By any chance, are you afraid of centaurs, beetles, and spiders?"`
+			`	"No, not at all. I'm not afraid of them. Why do you ask?"`
+			choice
+				`	"Because the Coalition are a race of three alien species, who look like spiders, beetles, and centaurs.`
+			`	"Interesting. Their stories might just inspire my next novel. But, they wouldn't speak English. How are they going to read my books?"`
```

Comment:
Is there really an explicit policy against mentioning specific human languages within dialog? Regardless, I think I'll just change it to "our language" and let the player make assumptions on their own.

Reply:
Well the game doesn't ever refer to the language spoken by the player and characters as a specific one, I dunno if we want to make it canon that it's English just because it's all in English. If we did, that'd imply changing it to other languages like Portuguese or German if/when we get translations.